---
layout: page
title: Research
modified: 2020-01-01
comments: false
excerpt: "An overview of research interests and directions in our lab."
---

{% include _toc.html %}


The vast majority of life on our planet is *microbial*. An astonishing number of microbial organisms living in terrestrial and marine habitats represent a biomass that exceeds every living organism that can be seen by naked eye, combined.

The inconceivable diversity of microbes allow them to synthesize or break down a wide array of chemical substrates, and govern biogeochemical cycles that make Earth a habitable planet for much less talented organisms (such as ourselves). Our own body is also home to a diverse assemblage of microbial cells. Bacteria that colonize our gastrointestinal tract help us maintain our health by extracting energy from undigested carbohydrates, synthesizing vitamins, and metabolizing xenobiotics.

Microbes are essential to Earth’s functioning at every scale, and understanding them is imperative for a complete understanding of life.

In our lab, we combine our expertise and interest in microbiology and computation to investigate the diversity and functioning of microbial communities in environments ranging from the human gastrointestinal tract and oral cavity, to sewages, oceans, and soils.

We design experiments, develop software platforms, and work with high-throughput sequencing data for marker genes, metagenomes, and metatranscriptomes to get closer to understanding mechanisms by which microbes interact with their surroundings, evolve, disperse, and initiate and/or adapt to environmental change.

We believe in interdisciplinary, hypothesis-driven, open and collaborative science, and we strive to identify the best computational and experimental practices that follows the inspiration from biology.

The following is an incomplete list of current or future interests of our group.

<div style="clear:both"></div>

## Studying fundamentals of microbial life and evolution

Despite their great importance for the habitability of the planet and maintenance of our health, our understanding of the diversity, functioning, and evolution of microbial life is far from being complete.

This is partially due to our inability to bring microbial life into the lab environment for comprehensive, conventional investigations: in most cases, it is very challenging to successfully isolate individual microbes from their interactive and complex environments and keep them functionally alive in a controlled setting. Even when isolation is possible, understanding how well the isolated members of microbial populations represent their environmental population is not necessarily always clear. Thankfully, in parallel to conventional approaches, we can use state-of-the-art molecular and computational techniques to recover the genomic content of naturally occurring microbes directly from the environment, and investigate some of the most fundamental aspects of their life and evolution. Our lab often uses high-throughput sequencing data to identify known and novel populations of microbes, determine their distribution, functional potential, and genomic heterogeneity to study their relationships with other microbes and environments they reside, which can inform us about their evolution.

Microbial ecology and evolution is happening everywhere, and investigating fundamentals of microbial life is independent of habitats. Hence, we are as happy in our lab working with marine environments as we are working with the human guts, or insect ovaries.

<div style="width: 300px; float: right; padding-left: 20px;">
<a href="http://i.imgur.com/HrlXPOF.jpg"><img src="http://i.imgur.com/HrlXPOF.jpg" style="border:none;" /></a>
</div>

For instance, [one of our recent studies focused on SAR11](https://doi.org/10.1101/170639), which is one of the most abundant microbial lineages in marine habitats. In collaboration with [Stephen Giovannoni](http://microbiology.science.oregonstate.edu/dr-stephen-giovannoni) from the **Oregon State University**, [Mike Rappé](https://rappelab.wordpress.com/) from the **University of Hawai'i**, and [Ismail Uysal](https://www.researchgate.net/profile/Ismail_Uysal3) from the **University of South Florida**, we developed and applied novel approaches to investigate non-synonymous genomic heterogeneity that emerges in a *single SAR11 population* as it goes around the world through large oceanic current movements. Using single-amino acid variants, we observed significantly more protein variants in cold currents and an increased number of protein sweeps in warm currents, exposing a global pattern of alternating genomic diversity for this SAR11 population. From the geographic partitioning of SAAVs we could suggest that natural selection, rather than neutral evolution, is the main driver of the evolution of SAR11 in surface oceans. We will continue to explore approaches to link environmental heterogeneity to the genomic context and protein structure, and generate testable hypothesis from our 'omics studies to better understand determinants of microbial fitness and evolution.


## Metagenomic and experimental insights into Fecal Microbiota Transplantation experiments

Fecal Microbiota Transplantation (FMT) experiments, the quite literal transference of fecal matter from a healthy donor to a recipient, gained recognition as an effective and relatively safe treatment for recurrent or refractory *Clostridium difficile* infection (CDI). Its success in treating otherwise very hard to eradicate CDI sparked great interest in investigating FMT as a treatment also for other medical conditions associated with intestinal dysbiosis, such as ulcerative colitis, Crohn’s disease, irritable bowel syndrome, and even metabolic syndrome or neurodevelopmental and autoimmune disorders.

Despite the excitement due to its therapeutic potential, FMTs also present challenges for researchers and clinicians with potential adverse outcomes, including the transfer of infectious organisms or contaminants from the environment. Deeper insights into FMTs from a basic science perspective will likely help us address these caveats, while teaching us a lot about the microbial dynamics. In fact, in our lab we see FMTs as excellent tools to study the functional basis of human gut colonization and the microbial response to environmental stress. As we learn about microbes by studying FMTs, we are hoping to get closer to understand key microbial populations that are responsible for beneficial outcomes of this procedure.

<div style="width: 300px; float: right; padding-left: 20px;">
<a href="{{ site.url }}/images/fmt_figure_01.png"><img src="{{ site.url }}/images/fmt_figure_01.png" style="border:none;" /></a>
</div>

[In a recent pilot study](https://doi.org/10.1186/s40168-017-0270-x), we collected fecal samples from two recipients before and after an FMT procedure. By using our metagenomic approaches, we could track individual donor microbial populations in a very highly resolved manner in the recipient guts to study microbial colonization dynamics. Some donor microbes that were initially absent from the recipient guts colonized both individuals successfully, and we could detect them even 8 weeks after the transfer. In contrast, some other donor microbes failed to colonize neither of our recipients. Most interestingly, we found that the colonizers we could identify in our pilot study were common among a much larger cohort of healthy people, and non-colonizers occurred rarely in the same cohort. We use these crucial insights to investigate the functional potential of microbes with high colonization properties, inform cultivation experiments in our lab, and look forward to generate hypotheses that we can translate to model systems.

<div style="clear:both"></div>


If you have any questions, please don't hesitate to [write to us]({{site.url}}/people/).

